app/src/components/MUIModal/MessageModal/MessageModal.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import MUIModal from '../MUIModal'
+
+interface MessageModalProps {
+    handleClose: () => void
+    modalOpen: boolean
+    modalHeadingText: string
+    modalMessage: string
+}
+
+function MessageModal(props: MessageModalProps) {
+    const { handleClose, modalOpen, modalHeadingText, modalMessage } = props
+    return (
+        <MUIModal handleClose={handleClose} modalOpen={modalOpen}>
+            <h2 style={{ color: 'red' }}>{modalHeadingText}</h2>
+            <p>{modalMessage}</p>
```

Comment:
Somewhere around here should live a Button element allowing the user to acknowledge the Message, instead of forcing them to click outside the modal window or on the 'X' to make it disappear. We may want to keep the top-level handleClose() callback handy here since there is no conditional behavior associated with closing this modal; let's bring this up with UX